R/intervals.R

```diff
@@ -0,0 +1,625 @@
+#' @title Generate columns to help describe numeric variable intervals
```

Comment:
Can you add a function to go from, for example, "age" to "age_length" without having to go through "age_start" and "age_end"? In `lifetableUtils` I am mostly using "age" and in order to add "age_length" with these functions I had to rename "age" to "age_start", then use `gen_end`, `gen_length`, and rename "age_start" back to "age". Unless you think we should always be using "age_start" and "age_end".

Reply:
Yeah I made a comment about this in `lifetableUtils`. I think it should switch to using "age_start" and "age_end" everywhere